build.sbt

```diff
@@ -706,6 +707,23 @@ def beamRunnerSettings: Seq[Setting[_]] = Seq(
   libraryDependencies ++= beamRunnersEval.value
 )
 
+def vectoredReadSettings: Seq[Setting[_]] = sys.props
+  .get("vectoredReadsEnabled")
```

Comment:
So if I understand things correctly to use vectored reads you need to have the `vectoredReadsEnabled` set at build time and also have `parquet.hadoop.vectored.io.enabled=true` in conf ?
Could you add doc in this PR ?

Reply:
the `vectoredReadsEnabled` setting isn't user-facing - it's only for enabling the overrides to run scio-examples with (similar to how we support `sbt -DbeamRunners=DataflowRunner scio-examples/runMain ...`

I realize I did not include the config setting in the doc though, will add!